lib/datadog_backup/core.rb

```diff
@@ -27,9 +27,10 @@ def backup
     # Returns the diffy diff.
     # Optionally, supply an array of keys to remove from comparison
     def diff(id)
-      current = except(get_by_id(id)).deep_sort.to_yaml
+      current = except(get_by_id(id))
+      current_yaml = current ? current.deep_sort.to_yaml : {}.to_yaml
```

Comment:
I feel like this {} initialization is redundant to the one that was introduced in line 90. I believe this logic should be expressed in a higher order function.